tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/security/TasklistURIs.java

```diff
@@ -23,7 +23,7 @@ public final class TasklistURIs {
   public static final String REST_V1_API = "/v1/";
   public static final String REST_V1_EXTERNAL_API = "/v1/external/**";
   public static final String NEW_FORM = "/new/**";
-  public static final String ALL_REST_VERSION_API = "/v*/**";
+  public static final String ALL_REST_VERSION_API = "/v1/**";
```

Comment:
I guess this is not part of the desired changes @nathansandi

Reply:
Ups, accident push